### PR TITLE
Wilor daggr rewrite

### PR DESCRIPTION
**Pull request by pablo-rerun**
## Summary

This PR turns `wilor-nano` into a more complete and reviewable example package by splitting the original PyTorch inference path from a new optimized TensorRT video path, tightening the model/API types, and adding small Rerun/Gradio/Daggr flows that exercise the package as reusable nodes.

## What Changed

- Added ONNX export and TensorRT build helpers for the full post-crop WiLoR model and the raw detector graph.
- Added static-batch TensorRT runtime wrappers with reusable CUDA input/output buffers, static-batch padding, named tensor validation, and strict CUDA/dtype checks.
- Added a batched CUDA/TensorRT video pipeline that keeps decode, detector preprocessing, NMS, crop generation, and WiLoR inference on GPU before logging columnar Rerun outputs.
- Added GPU-native WiLoR crop generation with `torch.grid_sample`, including support for mirrored left-hand crops.
- Refactored the regular WiLoR model path toward clearer typed outputs and replaced the local MANO wrapper code with the shared SimpleCV MANO layer.
- Added simple Gradio/Rerun detection and keypoint nodes plus a Daggr wiring script for composing the two-step demo.
- Added RRD comparison tooling and a 30-frame reference recording so candidate TensorRT output can be compared against a checked-in Rerun recording with tolerance-based Arrow table comparisons.
- Expanded the WiLoR README with the new task surface, TensorRT artifact workflow, and RRD comparison workflow.
- Wired new Pixi tasks for `video-trt`, `export-onnx`, `build-trt`, and `compare-reference`, along with the ONNX/TensorRT dependencies needed by the WiLoR environment.

## Review Notes

- TensorRT `.trt` engines are intentionally treated as machine-local artifacts. The portable flow is to export ONNX and rebuild engines on the target GPU.
- The optimized path is intentionally separate from the original PyTorch path so reviewers can compare behavior and so the baseline remains easy to run.
- `monorepo-modularity-presentation.html` is a standalone presentation artifact included with the branch.

## Validation Coverage

This PR adds or updates focused tests for:

- TensorRT runtime import behavior, static-batch padding, oversized batch rejection, and CUDA input validation.
- ONNX export and TensorRT build configuration behavior using test fakes.
- TensorRT video API configuration, output grouping, and video-only pipeline wiring.
- GPU crop generation against the expected OpenCV-style crop semantics.
- Batched TensorRT pipeline preprocessing, crop handling, and empty-detection behavior.
- RRD schema/value comparison behavior.
- Simple keypoint JSON round-tripping and video batch pipeline behavior.

Useful local validation commands:

```bash
pixi run -e wilor-dev --frozen pytest -q packages/wilor-nano/tests
pixi run -e wilor-dev --frozen ruff check packages/wilor-nano
pixi run -e wilor-dev --frozen pyrefly check packages/wilor-nano
pixi run -e wilor --frozen video-trt --max-frames 30 --rr-config.save /tmp/wilor_candidate.rrd --rr-config.headless
pixi run -e wilor --frozen compare-reference
```

GitHub currently reports no checks on the `wilor-daggr-rewrite` branch.

### CODE REVIEW

File: pixi.toml [hunk 1]
```diff
@@ -348,12 +348,15 @@ python = "3.12.*"
 dill = ">=0.4.0,<0.5"
 trimesh = "*"
 pyrender = "*"
+torchvision = ">=0.25,<0.26"
+torchcodec = ">=0.10.0,<0.11"
 
 [feature.wilor.pypi-dependencies]
 wilor_nano = { path = "packages/wilor-nano", editable = true }
 daggr = { path = "/home/pablo/0Dev/personal/daggr", editable = true }
```
[the remark this answers is not in the document]
Comment (pablo-rerun) [reply pixi.toml:356]: Leaving this one intentionally unchanged for this PR. The portability concern is valid, but this branch needs the local Daggr checkout until https://github.com/gradio-app/daggr/pull/83 lands or is otherwise consumable. Pointing Pixi at the current upstream state would require building the JavaScript frontend locally, which is not practical for this PR. We should replace this with a pinned portable reference once that upstream fix is available.

File: packages/wilor-nano/src/wilor_nano/pipelines/wilor_hand_pose3d_estimation_pipeline_trt.py [hunk 2]
```diff
@@ -0,0 +1,428 @@
...
+        if len(frame_indices_for_crop) == 0:
+            return records, None
+
+        frame_indices: Int[Tensor, "n"] = torch.cat(frame_indices_for_crop)
+        centers_xy: Float[Tensor, "n 2"] = torch.cat(centers_for_crop)
+        box_sizes: Float[Tensor, "n"] = torch.cat(box_sizes_for_crop)
+        flips: Bool[Tensor, "n"] = torch.cat(flips_for_crop)
+        patches: Float[Tensor, "n 256 256 3"] = generate_rgb_image_patches_torch(
+            frames_rgb,
+            frame_indices=frame_indices,
```
[the remark this answers is not in the document]
Comment (pablo-rerun) [reply packages/wilor-nano/src/wilor_nano/pipelines/wilor_hand_pose3d_estimation_pipeline_trt.py]: Fixed in `66c96c1`: renamed the loop-local tensor to `frame_indices_for_frame` and the concatenated tensor to `all_frame_indices`, so the two scopes/semantics are no longer conflated. `pyrefly check` reports 0 errors for the touched pipeline file.
